### PR TITLE
add know issue part for README.md

### PR DESCRIPTION
**Pull request by neo-liang-sap**
**What this PR does / why we need it**:
add know issue part for README.md
**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:
```improvement operator
add know issue part for README.md
```

### CODE REVIEW

**Comment by gardener-robot-ci-1** [comment]
Thank you @neo-liang-sap for your contribution. I will start a build for your PR. Once started, the build URL will be posted here.

**Comment by gardener-robot-ci-2** [comment]
A build of this pull request has started. You can check on its progress here: https://concourse.ci.gardener.cloud/teams/gardener/pipelines/ops-toolbelt-master/jobs/master-pull-request-job/builds/1

File: README.md [hunk 1]
```diff
@@ -72,3 +72,9 @@ To build all pre-configured images run:
 ```bash
 $ .ci/build
 ```
+
+### Known issues
+1. Currently there's an know issue for switching to `/bin/sh`, we implemented an color scheme and also added some helper function to display in `/bin/bash` terminal which doesn't work in `/bin/sh`, as workaround when you want to use some script which by defailt needs to utilize `/bin/sh` please use following command to force it utilize `/bin/bash`: (take `chroot` for example)
```
Comment (petersutter) [inline README.md:77]: some minor changes and typo fixes:
```suggestion
1. Currently there's a known issue when using `/bin/sh`. We implemented a color scheme and also added some helper function to display in `/bin/bash` terminal which doesn't work in `/bin/sh`. As workaround when you want to use some script which by default needs to utilize `/bin/sh` please use `/bin/bash` instead if possible: (take `chroot` for example)
```

**Comment by gardener-robot-ci-3** [comment]
Thank you @neo-liang-sap for updating the PR. I will start a build. Once started, the build URL will be posted here.

**Comment by neo-liang-sap** [comment]
thanks @petersutter

**Comment by gardener-robot-ci-1** [comment]
A build of this pull request has started. You can check on its progress here: https://concourse.ci.gardener.cloud/teams/gardener/pipelines/ops-toolbelt-master/jobs/master-pull-request-job/builds/2

**Comment by gardener-robot-ci-2** [comment]
Thank you @neo-liang-sap for updating the PR. I will start a build. Once started, the build URL will be posted here.

**Comment by gardener-robot-ci-1** [comment]
A build of this pull request has started. You can check on its progress here: https://concourse.ci.gardener.cloud/teams/gardener/pipelines/ops-toolbelt-master/jobs/master-pull-request-job/builds/3

**Pull request merged**